### PR TITLE
Clean code that shows the route in RouteDetail+ContentViewController.

### PR DESCRIPTION
**Pull request by danielVebman**
## Overview

Previously, the code to show the route was called every time the user's location updated, checking if it was the first time or not. I removed that code and instead show the route once at the end of `viewDidLoad`.

## Changes Made

Removed old functions/variables and replaced them with calls in `viewDidLoad`. After talking to the design people, the map now immediately shows the zoomed-in route instead of animating the zoom-in.

I also cleaned some miscellaneous code (only style, not substance) and updated the CocoaPods version.

## Test Coverage

Manually tested in simulator with feedback from designers.

## Related PRs or Issues (delete if not applicable)

Solution to issue #267 and continuation of PR #324.

### CODE REVIEW

File: TCAT/Controllers/RouteDetail+ContentViewController.swift [hunk 1]
```diff
@@ -74,6 +73,14 @@ class RouteDetailContentViewController: UIViewController {
 
         // Debug Function
 //        createDebugBusIcon()
```
Comment (kevinchan159) [inline TCAT/Controllers/RouteDetail+ContentViewController.swift]: fix indenting

File: TCAT/Controllers/RouteDetail+ContentViewController.swift [hunk 2]
```diff
@@ -84,45 +91,39 @@ class RouteDetailContentViewController: UIViewController {
 
         let isWalkingRoute = directions.reduce(true) { $0 && $1.type == .walk }
         // Plot the paths of all directions
-        for (arrayIndex, direction) in directions.enumerated() {
+        for (directionIndex, direction) in directions.enumerated() {
```
Comment (kevinchan159) [inline TCAT/Controllers/RouteDetail+ContentViewController.swift:94]: nice refactoring!!!

**Review by kevinchan159 — APPROVED** [review]
nice work! love the refactoring :)

File: TCAT/Controllers/RouteDetail+ContentViewController.swift [hunk 3]
```diff
@@ -452,7 +438,7 @@ class RouteDetailContentViewController: UIViewController {
     }
 
     /** Draw all waypoints initially for all paths in [Path] or [[CLLocationCoordinate2D]], plus fill bounds */
-    private func drawMapRoute() {
+    func drawMapRoute() {
```
Comment (lucyxubroad) [inline TCAT/Controllers/RouteDetail+ContentViewController.swift]: why did you is this function no longer private? is drawmaproute being used anywhere else?

Comment (danielVebman) [reply TCAT/Controllers/RouteDetail+ContentViewController.swift]: Good catch! I had been using it in a different file but found a better way. It's back now.

File: TCAT/Controllers/RouteDetail+ContentViewController.swift [hunk 4]
```diff
@@ -73,7 +72,15 @@ class RouteDetailContentViewController: UIViewController {
         routeDetailViewController.navigationItem.setRightBarButton(shareButton, animated: true)
 
         // Debug Function
-//        createDebugBusIcon()
+        // createDebugBusIcon()
```
Comment (lucyxubroad) [inline TCAT/Controllers/RouteDetail+ContentViewController.swift:75]: Not sure what this is!

**Pull request merged**